### PR TITLE
#166111798 Get repayment transaction by ID

### PR DESCRIPTION
**Pull request by victor-shagor**
#### What does this PR do?
         Get repayment transaction by ID
#### Description of Task to be completed?
          build an endpoint for users to get repayment transaction history by ID
#### How should this be manually tested?
clone the repo and test locally
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166111798](https://www.pivotaltracker.com/story/show/166111798)
#### Questions:

### CODE REVIEW

**Comment by luqmanoop** [comment]
Nice work so far but did you **build a database** in this PR like your description says?

File: server/queries/loanQueries.js [hunk 1]
```diff
@@ -36,5 +36,17 @@ const loans = {
       });
     }
   },
+  getRepaymentById(req, res) {
```
Comment (luqmanoop) [inline server/queries/loanQueries.js:39]: I believe this method/function gets a single repayment (by ID) history

**Review by luqmanoop — CHANGES_REQUESTED** [review]
I do not think your PR description aligns with submitted code. Please see the comments I left and make the necessary adjustments.

File: server/queries/loanQueries.js (hunk 1 above)
Comment (victor-shagor) [reply server/queries/loanQueries.js:39]: yes, it does... do you want me to rename it?

**Comment by victor-shagor** [comment]
> I do not think your PR description aligns with submitted code. Please see the comments I left and make the necessary adjustments.

comment adjusted

**Comment by luqmanoop** [comment]
LGTM

**Pull request merged**